docs/api/api-index.rst

```diff
@@ -1,10 +1,78 @@
 
+.. py:module:: datatable
+
+datatable API
+=============
+
+=========================  =====================================================
+**Classes**
+--------------------------------------------------------------------------------
+:class:`Frame`             Main data table class
+:class:`ltype`
+:class:`stype`
+
+**Functions**
+--------------------------------------------------------------------------------
+:func:`fread()`            Read CSV/text/XLSX/Jay/other files
+:func:`iread()`            Same, but read multiple files at once
```

Comment:
"Same as `fread()`, ..." would sound better to me.

Reply:
ok